app/background.cc

```diff
@@ -183,11 +185,14 @@ int grab_background(std::shared_ptr<background_t> pbkd, int width, int height, c
     if (pbkd->video) {
         // grab frame & frame no. under mutex
         std::unique_lock<std::mutex> hold(pbkd->rawmux);
-        cv::resize(pbkd->raw, out, cv::Size(width, height));
+        cv::Rect_<int> crop = calcCropping(pbkd->raw.cols, pbkd->raw.rows, width, height);
```

Comment:
```suggestion
        cv::Rect crop = calcCropping(pbkd->raw.cols, pbkd->raw.rows, width, height);
```

Reply:
I have updates my cropping branch accordingly. Your suggestion is within my code